fhir-audit/src/main/java/com/ibm/fhir/audit/cadf/model/CadfEvent.java

```diff
@@ -765,50 +765,64 @@ public static void generate(CadfEvent obj, JsonGenerator generator)
             if (obj.getTags() != null) {
                 generator.writeStartArray("tags");
                 for (String tag : obj.getTags()) {
-                    generator.write(tag);
+                    if (tag != null) {
```

Comment:
Maybe you can use ObjectUtils.isNotEmpty to validade this obj

Reply:
I like ObjectUtils, but the intention here is to validate not null only, the generated(input) values are either null or with no-empty value, there is no empty value case.  so, I don't want to introduce a new maven dependency for it.